tracardi/process_engine/action/v1/time/time_difference/plugin.py

```diff
@@ -71,6 +76,25 @@ def register() -> Plugin:
                         description="Please provide path to date, which will be considered the reference date for "
                                     "calculations, and so called 'now' date, in one of available forms.",
                         fields=[
+                            FormField(
+                                id="reference_date_format",
+                                name="Format of reference date",
+                                description="This field defines type of date that is considered as the lower bound "
+                                            "of calculated time span. "
+                                            "Now requires 'now' value in reference date field. "
+                                            "Date requires date in reference date field "
+                                            "(Example: 2021-03-14). "
+                                            "Path requires path to wanted date (Example: event@metadata.time.insert).",
+                                component=FormComponent(type="select", props={
```

Comment:
Why there is a select form data format at all. If someone types date then it will be parsed if types now then we can filter this as well. When types dotted notation DotAccessor will pick it up. we do not need types here.